Update IntelliJ setup instructions with Gradle workaround (#14234)
JabRef/jabref: docs/getting-into-the-code/guidelines-for-setting-up-a-local-workspace/intellij-11-code-into-ide.md

# Step 1: Get the code into IntelliJ

## Pre-condition: Gradle caches filled

> On Linux, with IntelliJ IDEA 2025.2.4 (Ultimate Edition), there are issues with importing.

Author: Haven't tried yet. Can try on Monday.

The workaround is to run JabRef once from the command line.

- Linux: Execute `./gradlew :jabgui:run`
- Windows (Powershell): Execute `.\gradlew :jabgui:run`

In case Gradle does not find a JDK, use [`gg.cmd`](https://github.com/eirikb/gg) as follows:

1. Download <https://github.com/eirikb/gg/releases/latest/download/gg.cmd>

> 2. Move the file into your JabRef code

Reviewer: ```suggestion
2. Move the file to your JabRef project directory
```

3. Run the GUI

   - Windows: `.\gg.cmd gradle:java@24 jabgui:compileJava`
   - Linux: `sh -x ./gg.cmd gradle:java@24 jabgui:compileJava`

4. Wait several minutes until command completes.

>    On an Intel(R) Core(TM) i5-1350 (16 GB RAM), it took more than 15 minutes.

Reviewer: ```suggestion
4. Wait until the command execution completes.
   On an Intel(R) Core(TM) i5-1350 (16 GB RAM), it takes more than 15 minutes.
```
